src/main/webapp/js/stats-loader.js

```diff
@@ -0,0 +1,23 @@
+// Fetch stats and display them in the page.
```

Comment:
I'd suggest the following formulation:

"use strict";
(function(){
   // ... your code here ...
})();

This formulation will give you a bit more checking and will also ensure that functions/variables are encapsulated and private unless you explicitly export them (e.g. by assigning them to a property of the global `window` object).